Add "views their own" note to Membership.md (#2579)
OBOFoundry/OBOFoundry.github.io: docs/Membership.md

The OBO Foundry Operations Committee discusses, oversees, and ensures the completion of the fundamental day-to-day activities of the Foundry. The Committee is composed of three working groups. Anyone who is active in a working group (active being based on both attendance at WG meetings and actual work done for working groups) is considered a member of the Operations Committee.

> Note: all views expressed by members of the OBO Foundry Operations Committee are their own, and do not necessarily represent the opinion of the organisations they are affiliated with.

Author: ```suggestion
Note: all views expressed by individual members of the OBO Foundry Operations Committee are their own, and do not necessarily represent the opinions of the OBO Foundry as a whole nor of their affiliated organisations.
```
